generators/app/templates/src/app/screens/OnBoarding/index.js

```diff
@@ -0,0 +1,20 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import Routes from '@constants/routes';
+
+import './i18n';
+import Swiper from './components/Swiper';
+
+const OnBoardingContainer = ({ navigation: { navigate } }) => {
+  const skipOnBoarding = () => navigate(Routes.App);
+
+  return <Swiper skip={skipOnBoarding} />;
+};
```

Comment:
Use a function instead of an arrow function

Reply:
Why this?